docs/getting-started-guides/fedora/fedora_manual_config.md

```diff
@@ -151,3 +139,56 @@ fed-minion          <none>              Ready
 **The cluster should be running! Launch a test pod.**
 
 You should have a functional cluster, check out [101](https://github.com/GoogleCloudPlatform/kubernetes/blob/master/examples/walkthrough/README.md)!
+
+**Run time addition/deletion of minions (or nodes).**
```

Comment:
Does it really make sense to put this AFTER we say the cluster is up?

Shouldn't we remove the --machines= stuff from the controller-manager?